### ISSUE TITLE
Use uuid module instead of node-uuid

### ISSUE THREAD

**Issue by ddimitrioglo** [pwmckenna/node-travis-ci#20]
### Are you requesting a feature or reporting a bug?
This is an enhancements.

### What is the current behavior?
During npm install, receiving following warning:

> deprecated node-uuid@1.4.8: Use uuid module instead

### Specify your

* operating system: macOS High Sierra (10.13.1)
* travis-lint version: ^1.0.0
* node.js version: v6.11.4

Could you please upgrade the module, looks like it could be upgraded.

**Comment by ljharb** [comment]
Deprecation warnings like this aren't very helpful, and can always be safely ignored. Unfortunately the message doesn't explain why it's deprecated.

That said, a PR that switches this project to `uuid` would be appreciated.

**Comment by ddimitrioglo** [comment]
I see 2 PR's opened on May 19, 2016 and Nov 28, 2016, and I'm not sure that my PR will be merged, am I wrong? ;)

**Comment by ljharb** [comment]
@ddimitrioglo you are wrong, yes; outstanding PRs are unmerged because they're not mergeable yet.
